assets/src/edit-story/components/statusCheck/statusCheckFailed.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useCallback } from 'react';
+import { Plain } from '../button';
+import Dialog from '../dialog';
+import Link from '../link';
+import { trackClick } from '../../../tracking';
+import { TranslateWithMarkup } from '../../../i18n';
+
+const Paragraph = styled.p`
+  font-family: ${({ theme }) => theme.fonts.body1.family};
+  font-size: ${({ theme }) => theme.fonts.body1.size};
+  line-height: ${({ theme }) => theme.fonts.body1.lineHeight};
+  letter-spacing: ${({ theme }) => theme.fonts.body1.letterSpacing};
+`;
+
+const SUPPORT_URL = __(
+  'https://wordpress.org/support/plugin/web-stories/#new-topic-0',
+  'web-stories'
+);
```

Comment:
Not sure if this needs to be localizable, but now it is.

Reply:
Yup makes sense.

Perhaps without the fragment though

Reply:
I'll leave it as-is for now. If we learn something new, we can update it.